src/Caching/SqlServer/src/Microsoft.Extensions.Caching.SqlServer.csproj

```diff
@@ -1,9 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Description>Distributed cache implementation of Microsoft.Extensions.Caching.Distributed.IDistributedCache using Microsoft SQL Server.</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;PKG0001</NoWarn>
```

Comment:
Just curious: Where does this warning come from?

Reply:
Removing the `System.Data.SqlClient` dependency is considered (rightly so) a breaking change:

```
C:\Users\anurse\code\aspnet\Extensions\eng\targets\ResolveReferences.targets(140,5): error PKG0001: Package references changed since the last release. This could be a breaking change. References removed: [C:\Users\anurse\code\aspnet\Extensions\src\Caching\SqlServer\src\Microsoft.Extensions.Caching.SqlServer.csproj]
C:\Users\anurse\code\aspnet\Extensions\eng\targets\ResolveReferences.targets(140,5): error PKG0001:  - System.Data.SqlClient [C:\Users\anurse\code\aspnet\Extensions\src\Caching\SqlServer\src\Microsoft.Extensions.Caching.SqlServer.csproj]
```

Prior art (other projects in this repo) suggests that suppressing is the way to handle that.